### PR TITLE
Add manual testing

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -491,3 +491,85 @@ shows you the welcome screen for the financial planner app
    1. Type `exit` into the terminal. 
    2. Expected: the financial planner will exit with a goodbye message.
 Under the data newly created data directory, a watchlist.json and a data.txt file will be created
+3. WatchList Feature
```
Comment (NeoMinWei) [inline docs/DeveloperGuide.md:494]: should this be ###WatchList Feature instead

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -491,3 +491,85 @@ shows you the welcome screen for the financial planner app
...
+
+Now we can visualize these expenses using 3 different charts (pie/bar/radar)
+```
+vis /t expense /c pie
+vis /t expense /c bar
+vis /t expense /c radar
+```
+You can run the 3 commands separately to see different charts
+
+We can do the same the income
```
Comment (ryan1604) [inline docs/DeveloperGuide.md]: "the same to income" u mean?

**Review by ryan1604 — APPROVED** [review]
LGTM

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (wwweert123) [reply docs/DeveloperGuide.md:494]: help me change i merge liao

**Pull request merged**